contracts/pTokenNoGSN.sol

```diff
@@ -79,6 +80,11 @@ contract PTokenNoGSN is
             "Recipient cannot be the token contract address!"
         );
         _mint(recipient, value, userData, operatorData);
+        if (userData.length > 0) {
+            // TODO: check if this is needed, and, eventually, replace .code with a function
+            // require(recipient.code.length > 0, "Recipient is not a contract");
```

Comment:
This is a hangover from solidity v <= 0.3.6, where function calls to contracts that had no such function, or non-contracts, just immediately returned with no errors, hence this check.

Later solidity - as your test shows - reverts, making the code-size check pointless. 

So yes, feel free to remove.